### PR TITLE
Ft view articles 167064436

### PR DESCRIPTION
**Pull request by jabichris**
#### Description
This PR is for adding functionalities such as:
- View all articles functionality
- View one article functionality
- Delete one article if you are the author

#### Type of change
N/A

#### How has it been tested
By going to `http://localhost:3002/articles` for all articles and `http://localhost:3002/articles/:slug` for one article.

 #### Pivotal tracker story ID
#167064436
#### Screenshots
Screen Shot 2019-07-16 at 20 41 16
If you are the author of the article, This is how you will see your article.

Screen Shot 2019-07-16 at 20 27 18
If you are not the author of the article, This is how you will see your article.

Screen Shot 2019-07-16 at 20 26 22
This is how you will see all articles.

### CODE REVIEW

**Review by Niyitangasam — COMMENTED** [review]
@jabichris , I am not able to test this feature on Heroku deployment.

**Review by Kabalisa — CHANGES_REQUESTED** [review]
@jabichris. on Heroku, an article is being rendered with HTML tags that it is stored as in the database. can you change that.

**Comment by jabichris** [comment]
@Kabalisa, I worked on the changes you requested. You can Re-review

**Review by Niyitangasam — APPROVED** [review]

**Review by Kabalisa — APPROVED** [review]

File: src/components/articles/articleCard.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { PropTypes } from "prop-types";
+import Moment from 'react-moment';
+import 'moment-timezone';
+import Image from '../../assets/images/pexels-photo-46710.jpeg'
+
+const ArticleCard = ({ article, onClick }) => {
+    const dateToFormat = article.createdAt;
+  return (
```
Comment (macphilips) [inline src/components/articles/articleCard.jsx:9]: please format the file properly

File: src/actions/delete.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import http from "../helpers/httpServices";
+import { DELETE_STORY } from "./types";
+
+export const deleteOneStory = (slug) =>  dispatch => {
```
Comment (macphilips) [inline src/actions/delete.js]: Please combine these files into one: `src/actions/delete.js`, `src/actions/oneStory.js`, `src/actions/viewArticles.js`

File: src/components/common/loader.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import animation from '../../assets/images/animation.gif'
+
+class Loader extends Component{
```
Comment (macphilips) [inline src/components/common/loader.jsx]: Please apply proper formatting to this file
Also, make this a functional component

**Pull request merged**